src/net/wooga/jenkins/pipeline/TestHelper.groovy

```diff
@@ -5,7 +5,7 @@ import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
 /**
 * Creates a step closure from a unity version string.
 **/
-def transformIntoCheckStep(platform, testEnvironment, coverallsToken, config, checkClosure, finallyClosure, skipcheckout = false) {
+def transformIntoCheckStep(platform, testEnvironment, coverallsToken, config, checkClosure, finallyClosure, skipcheckout = false, errorHandler) {
```

Comment:
as this is the last paramter and `skipcheckout` already has a default value `errorHandler` needs one too.
I think we could also write a new overload which gets called with a `null` `errorHandler`